cmd/argocd/commands/admin/settings.go

```diff
@@ -628,5 +648,7 @@ argocd admin settings resource-overrides action /tmp/deploy.yaml restart --argoc
 			})
 		},
 	}
+
+	command.Flags().StringArrayVar(&resourceActionParameters, "param", []string{}, "Action parameters (e.g. --param key1=value1)")
```

Comment:
Arrays should be plural "--params"

Reply:
Is it maybe a repeated flag? Like `--param key1=value2 --param key2=value2`?

Reply:
Yup, I looked too fast with the current usage of `StringArrayVar` and there is a mix of both, but the majority seems to be singular.